frontend/package.json

```diff
@@ -92,6 +92,7 @@
     "react-jsonschema-form": "^1.0.4",
     "react-lightweight-tooltip": "1.x",
     "react-linkify": "^0.2.2",
+    "react-measure": "^2.3.0",
```

Comment:
We should align on how we handle this. Currently console uses this custom hook:

https://github.com/openshift/console/blob/master/frontend/public/components/utils/ref-width-hook.ts

We might look at something based on hooks like:

https://github.com/rehooks/component-size

There are also polyfills for `ResizeObserver`:

https://caniuse.com/#search=ResizeObserver
https://github.com/que-etc/resize-observer-polyfill

@christianvogt mentioned dev console needs it for topology view as well.

cc @TheRealJon

Reply:
I tried `useRefWidth` and it works fine for me. `react-measure` dep is gone.